docs/pages/user-guide/tensor-and-indices.md

```diff
@@ -2,24 +2,75 @@
 
 ## Abstract Indices
 
+Abstract indices are symbolic objects that FunFact uses to denote how
+computations should happen between tensors. They can be created using the
+[`index`](../../../api/index_) and [`indices`](../../../api/indices)
+methods.
+
+``` py title="Examples"
+import funfact as ff
+i = ff.index('i')          # named index
+j = ff.index()             # anonymous index
+p, q = ff.indices('p, q')  # batch declaration
+r, s, t = ff.indices(3)    # batch declaration of anonymous indices
+```
 
 ## Abstract Tensors
 
-Tensors can be created using the [tensor](funfact.tensor) method.
+Tensors can be created using the [`tensor`](../../../api/tensor) method.
+These tensors are *abstract* in the sense that they only contain attributes
+such as dimensionality, shape, and optionally a symbol.
+Abstract tensor are not populated with numerical elements until we start to
+solve a concrete factorization problem.
 
-``` py
+``` py title="Examples"
 import funfact as ff
-a = ff.tensor('a', 5, 2, 4)                      # a random 3-way tensor
-```
-!!! note
-    The tensors that we create here are *abstract tensors*, which only have attributes such as dimensionality, shape, and a symbol. They have not been populated with numerical elements yet.
-
-```
 import numpy as np
+a = ff.tensor('a', 5, 2, 4)                      # a random 3-way tensor
 b = ff.tensor('b', np.arange(10).reshape(2, 5))  # a 2-way tensor (matrix)
 u = ff.tensor('u', 4)                            # a random vector
 v = ff.tensor(5)                                 # an anonymous random vector
 c = ff.tensor('c')                               # a scalar, i.e. 0-tensor
-i, j, k = ff.indices('i, j, k')                  # define indices
 ```
 
+## Special tensors
+
+FunFact is shipped with a small set of predefined special tensors due to their
+prevalence in linear algebra computations. These include:
+
+- The [unit tensor](../../../api/ones) of all 1s.
+- The [zero tensor](../../../api/zeros) of all 0s.
+- The [identity/Kronecker delta tensor](../../../api/eye). 
+
+## Initializers
+
+By default, the abstract tensors are populated with random elements draw
+from the normal distribution upon initialization of a factorization model. A
```

Comment:
Should we mention that by default the stdev of the distribution is very small?

Reply:
I was thinking about a future PR that relocates the docstrings for the initializers from class level to `__init__`. This way, the default values will be correctly presented on the API page. And then we can have a 'sharp bits' page to consolidate this type of information.